feature/stn/aggregate/index.js

```diff
@@ -115,9 +115,18 @@ ee.on('feat-stn', function () {
   registerHandler('bstXhrAgg', storeXhrAgg)
   registerHandler('bstApi', storeSTN)
   registerHandler('errorAgg', storeErrorAgg)
+  registerHandler('pvtAdded', processPVT)
 })
 
-function storeTiming (_t) {
+function processPVT (name, value, attrs) {
+  var t = {}
+  var owner = name === 'load' ? 'window' : 'document'
```

Comment:
I don't recall if the owner value is significant somehow for the backend. We might want to check with the visualization team. If yes, and the distinction between window and document is important, then we should probably also use window for `unload`. I am not sure about the others (fp, fcp, fi, lcp, pageHide).

Reply:
this has been deemed insignificant by dempla